packages/editor-ui/src/components/MainHeader/MainHeader.vue

```diff
@@ -184,15 +187,16 @@ async function navigateToExecutionsView(openInNewTab: boolean) {
 				@update:model-value="onTabSelected"
 			/>
 		</div>
-		<div class="github-button">
+		<div class="github-button" v-if="showGitHubButton">
```

Comment:
can we keep it? Would be great if we can keep e2e tests as close to reality of what users experience as possible

Reply:
I think we should bring it back once/if we decide to move this to a place where it's not so obstructive. In it's current location it's messing with too many e2e tests for my liking.

Reply:
also, for us to keep this, we need a wider viewport (which also doesn't always work). To keep the user experience as close to reality as possible, we should also not be assuming a viewport width of `1920px` either. 
Even I don't have that wide of a viewport 
![image](https://github.com/user-attachments/assets/2ead3828-b47a-43ac-8d29-25049253a230)

Reply:
Yes, thank you for the viewport changes. Would be great to get that in.

I am still not sure about removing Github button. Based on the thread on Slack, it does not seem Product is interested in removing it.

Reply:
we are not removing the button for any other context besides e2e tests. and we probably can't revert the viewport changes, since the button is taking quite a significant horizontal space, which will likely lead to flaky tests.

Reply:
Ok. Maybe it should also be behind an env variable, but we can wait till users start asking for this.